<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDetailSection</class>
 <widget class="QWidget" name="ProjectDetailSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectDetailSection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="generalGroupBox">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QFormLayout" name="generalFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="notesLabel">
        <property name="text">
         <string>Notes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="notesPlainTextEdit">
        <property name="placeholderText">
         <string>Add notes...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QDialogButtonBox" name="generalButtonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="metadataGroupBox">
     <property name="title">
      <string>Metadata</string>
     </property>
     <layout class="QFormLayout" name="metadataFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="uuidLabel">
        <property name="text">
         <string>UUID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="uuidDisplayLabel">
        <property name="font">
         <font>
          <family>Consolas</family>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
